myspecialthing.csv

```diff
@@ -0,0 +1,374 @@
+comments,model_name,identifier,parent_model_name,parent_link_type,attribute_name,new_attribute_name,attribute_type,link_model_name,description,display_name,format_hint,restricted,read_only,options,match,attribute_group,hidden,unique
```

Comment:
Do we need this in the repo, or was this only for testing?

Reply:
Good call, removing.